terraform/modules/sns/variables.tf

```
@@ -0,0 +1,15 @@
+variable "sns_main_name" {
+
+}
+
+variable "sns_slack_name" {
+
+}
+
+variable "sns_main_endpoint" {
+
+}
+
+variable "sns_slack_endpoint" {
```

```suggestion
variable "cg_platform_slack_notifications_email" {
```

Just to make the variable more descriptive of how it is used